observability-lib/dashboards/k8s-resources/component_test.go

```diff
@@ -10,6 +11,42 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+var generate = flag.Bool("generate", false, "generate test output")
```

Comment:
`update` is a common name for this:
```suggestion
var update = flag.Bool("update", false, "update golden test files")
```

Reply:
Alright thanks for suggestion